core/server/models/user.js

```diff
@@ -554,6 +554,12 @@ User = ghostBookshelf.Model.extend({
                 return bcryptCompare(object.password, user.get('password')).then(function then(matched) {
                     if (!matched) {
                         return Promise.resolve(self.setWarning(user, {validate: false})).then(function then(remaining) {
+                            if (remaining === 0) {
```

Comment:
Case in point: https://github.com/TryGhost/Ghost/commit/b7a80eb5fd515dae92b57cce394a3a618b006171